api/src/models/survey-view-update.ts

```diff
@@ -0,0 +1,35 @@
+import { getLogger } from '../utils/logger';
+
+const defaultLog = getLogger('models/survey-view-update');
+
+/**
+ * Pre-processes GET /project/{projectId}/survey/{surveyId} survey data
+ *
+ * @export
+ * @class GetSurveyData
+ */
+export class GetSurveyData {
```

Comment:
shouldn't this be in /models/survey-view?

Reply:
its for both view and update 😄

Reply:
thats why it is in `view-update`